consent/strategy_default.go

```diff
@@ -702,24 +702,24 @@ func (s *DefaultStrategy) executeBackChannelLogout(ctx context.Context, r *http.
 
 	var execute = func(t task) {
 		defer wg.Done()
+		l := s.r.Logger().WithRequest(r).
+			WithField("client_id", t.clientID).
+			WithField("token", t.token).
```

Comment:
Please do not log any type of credentials or tokens. It is a security risk!

Reply:
Removed token logging